CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
-# Contributing
+#
```

Comment:
@raelgc I have no idea. The GitHub in-browser editor did that. Shall I make another commit to fix that, or?